.gitignore

```diff
@@ -1,8 +1,10 @@
 DejaVuSansMono.ttf
 build/
 bin/
+*.DS_Store
```

Comment:
mhm, shouldn't be here, added by accident, but it *could* stay as it does not hurt. You should decide about it.

Reply:
Let it stay. :) I do however feel like you should remove the .ttf from this list if you're touching it anyways.